.github/workflows/main.yml

```diff
@@ -335,6 +336,59 @@ jobs:
           path: /report/
           retention-days: 1
 
+  run_tests_no_compose:
```

Comment:
rebase needed? this should be already in main

Reply:
fixed